packages/react-admin-rte/package.json

```diff
@@ -25,7 +25,7 @@
         "prebuild": "rimraf lib",
         "build": "tsc --project ./tsconfig.json",
         "start": "tsc --project ./tsconfig.json --watch",
-        "tsc": "tsc --noEmit --skipLibCheck --project ./tsconfig.json && :",
+        "tsc": "../../node_modules/.bin/tsc --noEmit --skipLibCheck --project ./tsconfig.json && :",
```

Comment:
wieso brauchst das hier extra so? beim start und build braucht man das auch nicht?

Reply:
das stimmt
wenn ich aber  "npm run tsc" manuell im package folder (packages/xxx) ausführe braucht mans so.
Ich fand das ganz praktisch beim entwickeln wenn ich nur ein package type-checken kann.
Muss aber so nicht ins Repo, ich wollts mal hier zur Diskussion stellen.

Reply:
ok das is das problem, du darfst in dem projekt bitte nie etwas mit npm ausführen.
wenn dann immer nur mit yarn also in dem fall `yarn tsc` dann gehts. NPM checkt die workspaces ned deshalb geht das ned